firmware/thruna/BMS/Src/App/App_Accumulator.c

```diff
@@ -465,3 +486,27 @@ bool App_Accumulator_BalancingEnabled(struct Accumulator *const accumulator)
 {
     return accumulator->balance_enabled;
 }
+
+void App_Accumulator_UpdateAuxThermistorTemps(struct Accumulator *const accumulator)
+{
+    accumulator->aux_thermistor_temps[thermistor_channel] = accumulator->read_thermistor_temp();
+
+    if (thermistor_channel < (NUM_AUX_THERMISTORS - 1))
+    {
+        thermistor_channel++;
+    }
+    else
+    {
+        thermistor_channel = 0;
+    }
```

Comment:
This can be: 
```c
thermistor_channel++;
thermistor_channel %= NUM_AUX_THERMISTORS 
```

Reply:
I'm getting the sense someone has done a bit of leetcode...

But yes that would be much cleaner